singer-io/tap-quickbooks
# Testing/more streams

# Description of change
Add the following streams to as many of the feature tests as possible:
 - employees
 - items
 - vendors
 - customers

# Manual QA steps
 - Run tests locally.
 - Verify bookmark values and other key data points manually as needed.
 
# Risks
 - None, test changes only
 
# Rollback steps
 - revert this branch
